eslint/babel-eslint-tests/test/integration/eslint/verify.js

```diff
@@ -1689,7 +1689,7 @@ describe("verify", () => {
         return hasGlobal;
         }
       `,
-      { "newline-before-return": 1 },
+      { "newline-before-return": 1, "no-console": 1 },
```

Comment:
ESLint 9 throws a good warning here: Previously we injected inline comment `/* eslint-disable no-console */` without actually enabling the `no-console` rule first.